.build/Makefile

```diff
@@ -0,0 +1,30 @@
+# set GOOARCH based on ARCH
+ifeq ($(ARCH),x86_64)
+GOARCH=amd64
+endif
+ifeq ($(ARCH),x86)
+GOARCH=386
+endif
+ifeq ($(GOARCH),)
+$(error "ARCH must be set to one of: x86, x86_64")
+endif
+
+# use "0.0.1" as default version if not supplied
+ifndef VERSION
+override VERSION=0.0.1
+endif
+
+.PHONY: package-googet
+package-googet: build pack-googet
+
+.PHONY: build
+build:
+	GOOS=windows go build -o ./bin/otelopscol_windows_$(GOARCH).exe ./cmd/otelopscol
+
+.PHONY: pack-googet
+pack-googet:
+	# googet doesn't support variable replacement or comamand line args so just use sed
+	sed 's/{{$$VERSION}}/$(VERSION)/g;s/{{$$ARCH}}/$(ARCH)/g;s/{{$$GOARCH}}/$(GOARCH)/g' \
```

Comment:
`sed` works, but `envsubst` seems to be more common / safer way to substitute variables (if your base Docker image supports it).
https://unix.stackexchange.com/a/294400

Reply:
👍

I did have to add `gettext-base` to the build docker image annoyingly, but I think that's fine.